packages/ods-react/src/components/tabs/src/components/tab/Tab.tsx

```diff
@@ -0,0 +1,30 @@
+import { Tabs } from '@ark-ui/react/tabs';
+import classNames from 'classnames';
+import { type ComponentPropsWithRef, type FC, type JSX } from 'react';
+import style from './tab.module.scss';
+
+export interface TabProp extends ComponentPropsWithRef<'button'> {
+  value: string,
+}
+
+const Tab: FC<TabProp> = ({
+  children,
+  className,
+  disabled,
```

Comment:
either add it specifically to the interface, or rely on the native button one in props (then you can remove it here and in the component JSX)